src/components/Fund/AllocatedFunds/AllocatedFundsContainer.js

```diff
@@ -0,0 +1,59 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Icon } from '@folio/stripes/components';
+
+import { fundsResource } from '../../../common/resources';
+import AllocatedFunds from './AllocatedFunds';
+
+const AllocatedFundContainer = ({ fundIds, mutator, labelId }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [allocatedFunds, setAllocatedFunds] = useState('');
+
+  useEffect(() => {
+    if (fundIds.length) {
+      setIsLoading(true);
+      mutator.allocatedFunds.GET({
+        params: {
+          query: fundIds.map(id => `id==${id}`).join(' or '),
```

Comment:
what if there is a 100 funds?

Reply:
added `batchFetch` from `stripes-acq-components`